Misc/NEWS.d/next/Library/2024-11-27-23-13-21.gh-issue-127335.IiqYeX.rst

```diff
@@ -0,0 +1 @@
+Add the latest ``termios`` macro constants to the :mod:`termios`.
```

Comment:
This NEWS entry does not communicate what's added. Either a) spell out the change so it makes sense in a changelog context, or b) remove the NEWS entry.